internal/runtime/init/init_test.go

```diff
@@ -183,6 +184,144 @@ p = true { 1 = 2 }`
 	}
 }
 
+func TestLoadTarGzsInBundleAndNonBundleMode(t *testing.T) {
```

Comment:
You'll need to add a test case for when there's a tar ball provided and a regular directory where the `-b` flag was not provided. See my comment above.

Reply:
Yes, if you see the 2nd test case in this method, the note says "load multiple bundles. one tar.gz and one folder. Bundle mode is false".  Can you clarify more if thats not correct.